cloud-controller-manager/do/loadbalancers.go

```diff
@@ -290,12 +339,9 @@ func (l *loadBalancers) UpdateLoadBalancer(ctx context.Context, clusterName stri
 		return err
 	}
 
-	_, _, err = l.resources.gclient.LoadBalancers.Update(ctx, lb.ID, lbRequest)
-	if err != nil {
-		return fmt.Errorf("failed to update load-balancer with ID %s: %s", lb.ID, err)
```

Comment:
The intention here is to annotate the error because the upstream error itself does not provide enough context (at least it didn't last time I checked).

Reply:
I replaced the `l.resources.gclient.LoadBalancers.Update` and subsequent error check with a call to `l.getLoadBalancer` which [returns an identically-formatted error message when the update fails](https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/274/files#diff-0f171474489b3a27add24a94c0da1505R321)